### PR TITLE
Feature/backend micro services

### PR DESCRIPTION
**Pull request by androsanta**
Added the following endpoints:
get /services
get /services/{name} (return list of service names)
delete /services/{name}

Currently get /services return the full JsonServerless object, should we filter some information?

### CODE REVIEW

File: packages/restlessness-cli/rln-backend/src/endpoints/delete-services-by-name/index.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import 'module-alias/register';
+import { LambdaHandler } from '@restlessness/core';
+import handler from './handler';
+import validations from './validations';
+
+export default LambdaHandler.bind(this, handler, validations, 'deleteServicesByName');
```
Comment (antoniogiordano) [inline packages/restlessness-cli/rln-backend/src/endpoints/delete-services-by-name/index.ts:6]: I think we use a custom handler for rln backend

File: packages/restlessness-cli/rln-backend/src/endpoints/get-services-by-name/index.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import 'module-alias/register';
+import { LambdaHandler } from '@restlessness/core';
+import handler from './handler';
+import validations from './validations';
+
+export default LambdaHandler.bind(this, handler, validations, 'getServicesByName');
```
Comment (antoniogiordano) [inline packages/restlessness-cli/rln-backend/src/endpoints/get-services-by-name/index.ts:6]: same here

File: packages/restlessness-cli/rln-backend/src/exporter.ts [hunk 3]
```diff
@@ -1,17 +1,20 @@
...
-import getModels from './endpoints/get-models';
-import getDaos from './endpoints/get-daos';
-import getEnvs from './endpoints/get-envs';
-import postEnvs from './endpoints/post-envs';
-import getOpenapi from './endpoints/get-openapi';
-import getPlugins from './endpoints/get-plugins';
-import getAuthorizers from './endpoints/get-authorizers';
-import getEndpointsById from './endpoints/get-endpoints-by-id';
-import deleteEndpointsById from './endpoints/delete-endpoints-by-id';
+import postEndpoints from 'root/endpoints/post-endpoints';
```
Comment (antoniogiordano) [inline packages/restlessness-cli/rln-backend/src/exporter.ts:2]: I think we should remove module alias from rln backend

File: packages/restlessness-cli/rln-backend/src/models/Service/index.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  static async getService(serviceName) {
+    await JsonServices.read();
+    return JsonServices.services[serviceName];
+  }
+
+  static async getServiceNameList(): Promise<string[]> {
+    await JsonServices.read();
+    return Object.keys(JsonServices.services)
+      //@TODO show offline-service ui-side?
+      .filter(s => s !== JsonServices.OFFLINE_SERVICE_NAME);
```
Comment (antoniogiordano) [inline packages/restlessness-cli/rln-backend/src/models/Service/index.ts:30]: we should hide offline and shared as well, devs can use only services created by themselves

**Review by antoniogiordano — CHANGES_REQUESTED** [review]

**Comment by androsanta** [comment]
Should we return the full JsonServerless object (for get /services/{name}) or only some information?

**Comment by antoniogiordano** [comment]
All the object is fine

**Comment by androsanta** [comment]
👍🏽 im doing the post /services

**Pull request merged**